ExomeDepth/run_ExomeDepth.py

```diff
@@ -232,6 +232,41 @@ def multiprocess_call(multiprocess_list):
     os.system(action)
 
 
+def return_vcf_stats(vcf):
+    stats = (subprocess.getoutput("tail -n1 {}".format(vcf)).split()[-1]).split(":")
```

Comment:
Waarom de `(` voor subprocess?

Reply:
Wat gebeurt er als de VCF eindigt met een lege regel? Ik zou het netter vinden als je de vcf gewoon opent in python en op zoek gaat naar de eerste/laatste/n vcf regel.

PyVCF is toch al een requirement.